scalyr_agent/tests/util/json_util_test.py

```diff
@@ -109,6 +116,18 @@ def __runtest(library):
                 )
                 obj3 = util.json_decode(text)
                 self.assertEquals(obj3, obj)
+
+                # Sanity test to ensure curly brace is always the last character when serializing
+                # a dict
```

Comment:
I would explicitly say something like "our rewind logic in the scalyr client relies on this."

Reply:
Done - ce5154dcd0e316a3ad3803983a332e56c23e2667.